### PR TITLE
make it possible to configure proceeds address

### PR DESCRIPTION
**Pull request by sparrowDom**
make it possible to configure proceeds address

Contract change checklist:
  - [x] Code reviewed by 2 reviewers. See [template](https://github.com/OriginProtocol/security/blob/master/templates/Contract-Code-Review.md) and [example](https://github.com/OriginProtocol/security/blob/master/templates/Contract-Code-Review-Example.md).
  - [x] Unit tests pass
  - [x] Slither tests pass with no warning
  - [ ] Echidna tests pass if PR includes changes to OUSD contract (not automated, run manually on local)

### CODE REVIEW

**Comment by DanielVF** [comment]
Looks good, @sparrowDom. The only change I see that we need is that it does not need a separate deploy. We can just add the setRewardsProceeds to the previous deploy, since it has not run yet.

File: contracts/contracts/harvest/Harvester.sol [hunk 1]
```diff
@@ -391,17 +413,17 @@ contract Harvester is Governable {
...
+        uint256 rewardsProceedsShare = (usdtBalance * vaultBps) / 1e4;
 
         require(
             vaultBps > tokenConfig.harvestRewardBps,
-            "Address receiving harvest incentive is receiving more rewards than the vault"
+            "Address receiving harvest incentive is receiving more rewards than the rewards proceeds address"
         );
 
-        usdt.safeTransfer(vaultAddress, vaultShare);
+        usdt.safeTransfer(rewardProceedsAddress, rewardsProceedsShare);
```
Comment (tomlinton) [inline contracts/contracts/harvest/Harvester.sol:423]: Not really related to this PR, but we should make it possible to pass in `address(0)` here for _rewardTo and skip the incentive logic and just transfer all the rewards to `rewardProceedsAddress`.

**Review by tomlinton — APPROVED** [review]
Looks good to me!

File: contracts/contracts/harvest/Harvester.sol (hunk 1 above)
Comment (tomlinton) [reply contracts/contracts/harvest/Harvester.sol:423]: Actually I mean if `_rewardTo == rewardProceedsAddress`

**Pull request merged**